File: iteration2/Assistant.java
import java.util.ArrayList;
import java.util.List;

class Assistant extends Staff {
    private String assistantID;

    // Constructors
    public Assistant() {

    }

    public Assistant(String name, String lastName, String username, String password, String assistantID, List<Student> studentList) {
        super(name, lastName);
        this.assistantID = assistantID;
    }

    public Assistant(String name, String lastName, String assistantID) {
        super(name, lastName);
        this.assistantID = assistantID;
    }

    public String getAssistantID() {
        return assistantID;
    }

    @Override
    public String toString() {
        return "Assistant: " + this.getFirstName() + " " + this.getLastName();
    }

//Assistant class, currently has no methods.
}

File: iteration3/Assistant.py
from Staff import Staff


class Assistant(Staff):
    def __init__(self, name="", last_name="", assistant_id=""):
        super().__init__(name, last_name)
        self.assistant_id = assistant_id

    def get_assistant_id(self):
        return self.assistant_id

    def __str__(self):
        return f"Assistant: {self.get_first_name()} {self.get_last_name()}"
